web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -304,6 +304,41 @@ fixmystreet.assets.bromley.unset_asset_owner = function() {
     $('#form_fms_layer_owner').val('');
 };
 
+fixmystreet.assets.bromley.park_asset_found = function(layer) {
+    var streets_category_groups = [
+        'Highway Enforcement',
+        'Highway Major Works',
+        'Road, Pavement and PROW Issues',
+        'Street Cleansing',
+        'Street Lighting and Road Signs'
+    ];
+    var selected = fixmystreet.reporting.selectedCategory();
+    if (streets_category_groups.includes(selected.group)) {
+        var $msg = $('<div class="box-warning"></div>');
+        $msg.append(
+            "We’ve noticed that you’ve selected a Streets category but that your map pin is located within" +
+            " a park, are the location and category correct? You can change category by going back or" +
+            " relocate your map pin by clicking and dragging.");
+        $msg.prependTo('.js-post-category-messages');
```

Comment:
On the example coords you provide, if you select e.g. Road, Pavement... click Continue and then a subcategory, the message appears twice (or one more time each category change).